README.md

```diff
@@ -423,84 +452,181 @@ layout_strategies.horizontal = function(self, max_columns, max_lines)
 end
 ```
 
-## Goals
+#### Command-line `WIP`
 
-### Pipeline Different Objects
+All `telescope.nvim` functions are wrapped in `vim` commands for easy access, its
+supports tab completions and settings options.
 
-(Please note, this section is still in progress)
+```viml
+:Telescope find_files |<tab> 
+:Telescope find_files prompt_prefix=🔍 
```

Comment:
For me completion works for builtins but not for opts. Am i missing something

Reply:
Sorry my bad, the new version should be clear. Yes completion for options feature will be awesome @glepnir don't you think so?

Reply:
yep. which opts doesn't work for you?

Reply:
I think it can be really useful for testing options. Also @glepnir for completion for builtins when I type hel and press tab to get help_tags, my input get replaced with the first builtin `get_files`. Is this normal?

Reply:
@glepnir i was thinking about that but it could be difficult with builtin specific options, like `show_all_buffers`, `word_match´, etc. But i think we can support options that work across builtins

Reply:
Someone can make a new issue for the completion for options. It's a good idea. It will require a bit of internal refactor, but it would be a good part of the refactor for generating vim docs.